viam-cartographer/src/carto_facade/carto_facade_test.cc

```diff
@@ -678,6 +700,19 @@ BOOST_AUTO_TEST_CASE(CartoFacade_demo) {
                    VIAM_CARTO_POINTCLOUD_MAP_EMPTY);
     }
 
+    {
+        viam_carto_get_internal_state_response isr;
+        BOOST_TEST(viam_carto_get_internal_state(vc, &isr) ==
+                   VIAM_CARTO_SUCCESS);
+        // special case: apparently the first call to AddSensorReading doesn't
+        // change the internal state, but subsequent calls do.
```

Comment:
Rewrite with neutral tone: "The first call to AddSensorReading doesn't change the internal state, but subsequent calls do."

Reply:
done